environment.yml

```diff
@@ -4,37 +4,28 @@ channels:
 dependencies:
   - python=3.8.13
   - apache-beam=2.40.0
-  - cdsapi=0.5.1
-  - pytest=7.2.0
-  - pytest-subtests=0.8.0
-  - cfgrib=0.9.10.2
-  - dask=2022.10.0
```

Comment:
We are using `dask` in weather-mv/regrid. Are we sure to remove it from here?

Reply:
Checked this. `xarray-beam==0.3.1` has `dask` in it's requirements and it will install `dask==2022.12.0`.
It is safe to remove this from here. 👍

Reply:
Thanks for checking, Deep! :)